### DISCUSSION TITLE
Verify verifiable presentation

### DISCUSSION THREAD

**Discussion by Jacopodd** [blockchain-lab-um/ssi-snap#651] [General]
@martines3000 
I'm doing an internship for university. 
Once I have created a verifiable presentation (with key did:key(EBSI)) how do I verify it?
Furthermore, the discord invitation link on the masca site is no longer valid.

**Comment by martines3000** [comment]
Hi. Here is another invite (this one does not expire): [Invite](https://discord.gg/M5xgNz7TTF).

Regarding the verification process:
1, You can use the "verifyData" RPC method. [docs](https://docs.masca.io/docs/integrate-masca/masca-connector#verify-data)
2. You can verify it on masca.io, here:
![image](https://github.com/blockchain-lab-um/masca/assets/35891136/ace78d2c-461b-452f-bfe1-d5a4d7deea57)

**Reply by Jacopodd** [reply]
Perfect, thank you, I will continue to write on the "Developer" section in discord
